casper/contracts/simple_casper.v.py

```diff
@@ -574,6 +581,8 @@ def withdraw(validator_index: int128):
 # Process a vote message
 @public
 def vote(vote_msg: bytes[1024]):
+    assert msg.sender == self.NULL_SENDER
```

Comment:
this might go to votable

Reply:
I would want `votable` to be an easy call on the client side that doesn't really care about actual tx details. It's truthiness should only depend on the `vote_msg` and the current state of the contract